tests/local_providers/titelive_stocks_test.py

```diff
@@ -90,13 +91,47 @@ def test_titelive_stock_provider_update_1_stock_and_1_offer(stub_get_stocks_info
     assert stock.available == 10
     assert Offer.query.count() == 1
 
+@freeze_time('2019-01-03 12:00:00')
+@clean_database
+@patch('local_providers.local_provider.feature_queries.is_active', return_value=False)
+@patch('local_providers.titelive_stocks.get_stocks_information')
+def test_titelive_stock_provider_always_update_the_stock_modification_date(stub_get_stocks_information, stub_feature_queries, app):
+    # Given
+    stub_get_stocks_information.return_value = iter([{
+        "ref": "0002730757438",
+        "available": 2,
+        "price": 4500,
+        "validUntil": "2019-10-31T15:10:27Z"
+    }])
+
+    offerer = create_offerer()
+    venue = create_venue(offerer, siret='77567146400110')
+
+    titelive_stocks_provider = get_provider_by_local_class('TiteLiveStocks')
+    titelive_stocks_provider.isActive = True
+    venue_provider = create_venue_provider(venue,
+                                           titelive_stocks_provider, is_active=True,
+                                           venue_id_at_offer_provider='77567146400110')
+    product = create_product_with_thing_type(id_at_providers='0002730757438')
+    offer = create_offer_with_thing_product(venue, product=product, id_at_providers='0002730757438@77567146400110')
+    yesterday = date.today() - timedelta(days=1)
```

Comment:
```suggestion
    yesterday = datetime.utcnow() - timedelta(days=1)
```

Je crois qu'on utilise jamais date.today() dans le reste de l'app